src/main/java/com/billing/pdfgenerator/domain/dto/Response.java

```diff
@@ -1,25 +1,18 @@
 package com.billing.pdfgenerator.domain.dto;
 
+import lombok.Builder;
 import lombok.Getter;
 import lombok.Setter;
 
 import java.time.LocalDateTime;
 
 @Getter
 @Setter
+@Builder
 public class Response {
 
     private String status;
     private LocalDateTime creationDate;
-    private byte[] pdf;
-
-    public Response(String status, LocalDateTime creationDate, byte[] pdf) {
-        this.status = status;
-        this.creationDate = creationDate;
-        this.pdf = pdf;
-    }
-
-    public Response() {
```

Comment:
Where is the constructor of **Response**? if you remove it from response you should use **@ NoArgsConstructor**